.gitignore

```diff
@@ -1,21 +1,21 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-**/node_modules
-
-# testing
-**/coverage
-
-# production
-**/build
-
-# misc
-**.DS_Store
-**.env
-**.env.local
-**.env.development.local
-**.env.test.local
-**.env.production.local
-**.idea
-
-**npm-debug.log*
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+**/node_modules
+
+# testing
+**/coverage
+
+# production
+**/build
+
+# misc
+**.DS_Store
+**.env
+**.env.local
+**.env.development.local
+**.env.test.local
+**.env.production.local
+**.idea
+
+**npm-debug.log*

```

Comment:
What are the changes here? Make sure that the line endings aren't changing (e.g. from `\n` to `\r\n`), this is a setting in Git.

Reply:
Thanks - changing the setting helped